### PR TITLE
build with linkchecker

### PR DESCRIPTION
**Pull request by ocefpaf**
This should fail badly b/c we never curated the links in this doc.

Fixed most of them but some are problematic b/c they are slow or I could not find a substitute:

- [x] `data_access_notebooks/2017-06-12-NCEI_RA_archive_history`: **broken** https://geo-ide.noaa.gov/wiki/index.php?title=ISO_Namespaces - HTTPSConnectionPool(host='geo-ide.noaa.gov', port=443): Max retries exceeded with url: /wiki/index.php?title=ISO_Namespaces (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7ffad7d8d950>: Failed to resolve 'geo-ide.noaa.gov' ([Errno -2] Name or service not known)"))
- [x] `data_access_notebooks/2017-06-12-NCEI_RA_archive_history`: **broken**    https://data.nodc.noaa.gov/geoportal/catalog/search/search.page - 404 Client Error: Not Found for url: https://www.ncei.noaa.gov/data/oceans/geoportal/catalog/search/search.page
- [x] `data_access_notebooks/2017-06-12-NCEI_RA_archive_history`: **broken**    https://www.nodc.noaa.gov/search/granule/catalog/searchtips/searchtips.page - 404 Client Error:  for url: https://www.ncei.noaa.gov/metadata/granule/geoportal/catalog/searchtips/searchtips.page
- [x] `data_management_notebooks/2020-12-08-DataToDwC`: **broken**    https://www.glos.us/code-sprint/ - HTTPSConnectionPool(host='www.glos.us', port=443): Max retries exceeded with url: /code-sprint/ (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ffad7d8f750>, 'Connection to www.glos.us timed out. (connect timeout=None)'))
- [x] `Video_Demo_Markdown/ioos`: **broken**    https://nccospublicstor.blob.core.windows.net/ioos/ioos_demo_1280.mp4 - 404 Client Error: The specified resource does not exist. for url: https://nccospublicstor.blob.core.windows.net/ioos/ioos_demo_1280.mp4

### CODE REVIEW

**Comment by ocefpaf** [comment]
@MathewBiddle I need your advice/help with the following links:

1.  There are two 404 and one 443 in the `2017-06-12-NCEI_RA_archive_history`. I could not find alternative links to fix those and. I wonder if we should rearrange the text to remove those links.
1. The `data_access_notebooks/2017-11-30-rerddap` is timing out on http://gcoos4.tamu.edu:8080/erddap, that server is notoriously slow... It is what it is and we can either find that dataset elsewhere, or archive the notebook, or accept the time out and skip the check there.
1. The link to https://www.glos.us/code-sprint/ in `2020-12-08-DataToDwC` is also timing out.
1. The IOOS video link https://nccospublicstor.blob.core.windows.net/ioos/ioos_demo_1280.mp4 is a 404.

**Comment by MathewBiddle** [comment]
Yes, let's rearrange the text in `2017-06-12-NCEI_RA_archive_history`.

The line with
> https://geo-ide.noaa.gov/wiki/index.php?title=ISO_Namespaces

can be removed.

All of the geoportal links can be adjusted to https://www.ncei.noaa.gov/metadata/geoportal/#searchPanel and https://www.ncei.noaa.gov/metadata/geoportal/#aboutPanel

The GLOS code sprint link should be https://glos.org/2019-code-sprint/

I have no clue what that video is 😢

File: jupyterbook/_config.yml [hunk 1]
```diff
@@ -35,6 +35,7 @@ parse:
     - substitution
 
   myst_url_schemes: [mailto, http, https] # URI schemes that will be recognized as external URLs in Markdown links
+  myst_dmath_double_inline: true  # Allow display math ($$) within an inline context
```
Comment (ocefpaf) [inline jupyterbook/_config.yml:38]: Note to self: This was the default but now it must be explicitly selected.

**Comment by ocefpaf** [comment]
Building the book with a linkchecker is passing now :tada: 

@MathewBiddle this one is ready to go. I remove that video BTW.

**Pull request merged**